src/CMakeLists.txt

```diff
@@ -45,3 +45,85 @@ ENDIF()
 set(ALL_EXE_TARGETS haploclique)
 
 install(TARGETS ${ALL_EXE_TARGETS} RUNTIME DESTINATION bin)
+
+set(libhaplo_SRCS
```

Comment:
Why are you adding headers to to this variable? This is unnecessary.

You could rather use
```
file(GLOB_RECURSE libhaplo_SRCS  "*.cpp")
```
and thus remove all cpp files from the executable, except for `haploclique.cpp`